rpcs3/Emu/RSX/rsx_cache.h

```diff
@@ -417,10 +419,126 @@ namespace rsx
 		std::string version_prefix;
 		std::string root_path;
 		std::string pipeline_class_name;
+		std::mutex fpd_mutex;
 		std::unordered_map<u64, std::vector<u8>> fragment_program_data;
 
 		backend_storage& m_storage;
 
+		std::string getMessage(u32 index, u32 processed, u32 entry_count) {
```

Comment:
brace on newline

Reply:
Also, prefer snake_case for member functions. I know this was inherited from a previous lambda name by another contributor.